app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorDetailView.java

```diff
@@ -70,6 +74,14 @@ private void setupButtons(View rootView) {
             cancelFavor();
           }
         });
+
+    chatBtn.setOnClickListener(
```

Comment:
should move this to the setupButtons() method

Reply:
It looks to me that it's already in the setupButtons() and there are no duplicates, am I wrong?

Reply:
Closing because it's already on setupButtons().